src/main/resources/beam-template.conf

```diff
@@ -134,6 +134,8 @@ beam.agentsim.agents.rideHail.repositioningManager.timeout = "int | 0"
 # Larger value increase probability of the ride-hail vehicle to reposition
 beam.agentsim.agents.rideHail.repositioningManager.demandFollowingRepositioningManager.sensitivityOfRepositioningToDemand = "double | 1"
 beam.agentsim.agents.rideHail.repositioningManager.demandFollowingRepositioningManager.numberOfClustersForDemand = "int | 30"
+beam.agentsim.agents.rideHail.repositioningManager.demandFollowingRepositioningManager.PercentageOfClosestClustersToConsider = "double | 0.2"
```

Comment:
Please follow convention for naming. Should be camel case with **lower case** for the first letter.

Also, this value is not a percentage, it's a fraction. Should be named "fractionOfClostesClustersToConsider"